bigframes/core/indexers.py

```diff
@@ -63,17 +64,14 @@ def __setitem__(self, key, value) -> None:
         index_column = block.index_columns[0]
 
         # if index == key return value else value_colum
-        block, insert_cond = block.apply_unary_op(
-            index_column, ops.partial_right(ops.eq_op, key)
-        )
-        block, result_id = block.apply_binary_op(
-            insert_cond,
-            self._series._value_column,
-            ops.partial_arg1(ops.where_op, value),
-        )
-        block = block.copy_values(result_id, value_column).drop_columns(
-            [insert_cond, result_id]
+        blcok, result_id = block.project_expr(
```

Comment:
`blcok` typo?

Reply:
good catch. fixed this and a few other bugs